odoo/addons/base/models/ir_ui_view.py

```diff
@@ -482,6 +482,8 @@ def _compute_defaults(self, values):
     @api.model_create_multi
     def create(self, vals_list):
         for values in vals_list:
+            if 'model' not in values and values.get('inherit_id'):
+                values['model'] = self.browse(values['inherit_id']).model
             if not values.get('type'):
                 if values.get('inherit_id'):
                     values['type'] = self.browse(values['inherit_id']).type
```

Comment:
remove this browse?

Reply:
What do you mean?
We need it to read the field from the parent 👀

Reply:
I mean there are two browse operations. Maybe a waste of time ?

Reply:
AFAIK `browse` isn't expensive at all, and they're both under two different `if` clauses so I'm not sure I'd change anything here 😅

In case I'm missing the obvious, could you use the GitHub suggestion feature?

Reply:
OK.that’s fine